src/optimizers/adadelta_optimizer.ts

```diff
@@ -119,4 +119,4 @@ export class AdadeltaOptimizer extends Optimizer {
     return new cls(config.learningRate, config.rho, config.epsilon);
   }
 }
-SerializationMap.register(AdadeltaOptimizer);
+registerClass(AdadeltaOptimizer);
```

Comment:
Lets make this serialization.registerClass.

I think its important that we don't lose the "serialization" is why we are registering this class. and not registering a class is not doing something larger in the context of the whole sytem.

I think this means we want to double import.
import serialization from ....
import {ConfigDict, Serializable, SerializableConstructor} from ...

We directly import the three base classes/interfaces because their name describes their purpose, and they are used so commonly that qualifiying their names gets messy/verbose very quickly.

I think the other option for "registerClass" is call it "registerSerializableClass" and then we could avoid the non-qualified import for it.

Reply:
I prefer to keep it called "registerClass" for succinctness. I re-revised the code so that they say `serialization.registerClass(...)`.